virtcontainers/factory/template/template.go

```diff
@@ -103,3 +116,13 @@ func (t *template) createFromTemplateVM() (*vc.VM, error) {
 
 	return vc.NewVM(config)
 }
+
+func (t *template) checkTemplateVM() error {
```

Comment:
This needs some comments what you are doing here and what those files mean.

Reply:
Will do.